samcli/cli/main.py

```diff
@@ -58,6 +59,18 @@ def print_info(ctx, param, value):
     ctx.exit()
 
 
+def print_version_option(ctx, param, value):
+    """
+    Since default implementation for click.version_option doesn't support callback's we need to create a simple version
+    of it, which will print version information, and check for new version and exit the cli
+    """
+    if not value or ctx.resilient_parsing:
+        return
+    click.echo("SAM CLI, version %(version)s" % {"version": __version__}, color=ctx.color)
+    inform_newer_version(True)
```

Comment:
minor: You should add the name of the parameter to make it more obvious what is `True`